gui/public/i18n/en/translation.ftl

```diff
@@ -366,18 +366,18 @@ onboarding-skip = Skip setup
 onboarding-continue = Continue
 onboarding-wip = Work in progress
 
-## WiFi setup
+## Wi-Fi setup
 onboarding-wifi_creds-back = Go Back to introduction
-onboarding-wifi_creds = Input WiFi credentials
+onboarding-wifi_creds = Input Wi-Fi credentials
 # This cares about multilines
 onboarding-wifi_creds-description =
-    The Trackers will use these credentials to connect wirelessly.
+    The Trackers will use these credentials to connect wireless.
```

Comment:
The Trackers will use these credentials to connect ``wireless``?

Reply:
ugh, I assumed IDEA caught an error and didn't double check smh 
![image](https://user-images.githubusercontent.com/20461725/211426408-0747209a-ec11-47bb-bb15-61583d3d0a28.png)
Thanks